enclave/src/rpc/author/mod.rs

```diff
@@ -200,7 +210,7 @@ where
         bytes_or_hash: Vec<hash::TrustedOperationOrHash<TxHash<P>>>,
         shard: ShardIdentifier,
         inblock: bool,
-    ) -> Result<Vec<TxHash<P>>> {
+    ) -> Result<Vec<TxHash<P>>> {        
```

Comment:
I suggest an automatic formatter to avoid whitespaces like this (sorry for the human linter again)

Reply:
No sorry necessary, added automatic removal of trailing whitespaces.